src/EnergyPlus/HVACFourPipeBeam.cc

```diff
@@ -1102,8 +1095,6 @@ namespace FourPipeBeam {
         using PlantUtilities::SetComponentFlowRate;
         using namespace std::placeholders;
 
-        bool dOASMode = false; // true if unit is operating as DOAS terminal with no heating or cooling by beam
-
```

Comment:
[src/EnergyPlus/HVACFourPipeBeam.cc:1105]:(style),[variableScope],The scope of the variable 'dOASMode' can be reduced.